README.md

```diff
@@ -733,7 +743,7 @@ ardrive upload-file -w wallet.json -F "6939b9e0-cc98-42cb-bae0-5888eca78885" --l
 ardrive upload-file -w wallet.json -F "6939b9e0-cc98-42cb-bae0-5888eca78885" --local-paths ./*.json
 ```
 
-### Name Conflict Resolution on Upload
+### Name Conflict Resolution on Upload<a id="conflict-resolution"></a>
```

Comment:
Is anything using the `conflict-resolution` anchor yet?

Reply:
Nope, I noticed it was missing and meant to add that here

Reply:
57562f5